jazz_test-lambda/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "test-lambda",
+  "version": "0.0.1",
+  "description": "Provides an API for Testing a lambda function and returns the Execution Status",

```

Comment:
"description": "API that triggers a function with a test event and returns function execution status/payload back",